lib/minitest/reporters/mean_time_reporter.rb

```diff
@@ -288,6 +325,58 @@ def rate(run, min, max)
         end
       end
 
+      # @return [Boolean] Whether the given :order option is valid.
+      def valid_order?
+        return true if orders.include?(options[:order])
+
+        fail Minitest::Reporters::MeanTimeReporter::InvalidOrder,
+             '`:order` option must be one of `:desc` or `:asc`'
```

Comment:
Though it is unlikely that list of orders will be changed but it would be great to use orders to create error message.